### PR TITLE
Support MBS v2.2.0

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: fedmsg.d/pdcupdater-example.py [hunk 1]
```diff
@@ -30,6 +30,9 @@
     # PkgDB details
     'pdcupdater.pkgdb_url': 'https://admin.fedoraproject.org/pkgdb',
 
+    # MBS details
```
Comment (mprahl) [inline fedmsg.d/pdcupdater-example.py]: Note we will have to configure this in Ansible

Comment (ralphbean) [reply fedmsg.d/pdcupdater-example.py]: Ack!

**Comment by ralphbean** [comment]
If travis is happy, I'm going to go ahead and merge this.

**Comment by ralphbean** [comment]
Matt is adding one more test.  :)

**Pull request merged**